### ISSUE TITLE
How can I calibrate my predicted ROI in the MMM with my geolift result ? 

### ISSUE THREAD

**Issue by elisakrammerfiverr** [google/lightweight_mmm#281]

**Comment by shubham223601** [comment]
@becksimpson, thanks for the idea.

 the things which is not clear to me is, how the below part of the code is used for model calibration here. Would be great if you can share some details Thank you
 
 ```
media_contribution = jnp.einsum(media_einsum, media_transformed, coef_media)
    observations = transform_kwargs.get('observations', None)
    if observations is not None:
      for i, obs in enumerate(observations):
        _ = numpyro.sample(
          name=f'observation_{i}',
          fn=dist.Normal(
            loc=media_contribution[
              obs['time_periods'],
              obs['channel']
            ].sum(axis=1),
            scale=0.2
          ),
          obs=jnp.array(obs['attributed_values'])
        )
```

**Comment by becksimpson** [comment]
channel_media_einsum = 'tc, c -> tc'

**Comment by nutterbrand** [comment]
@becksimpson In this implementation, is the following media_contribution, now used in the prediction calculation on on line #411?

`media_contribution = jnp.einsum(media_einsum, media_transformed, coef_media)
   observations = transform_kwargs.get('observations', None)
   if observations is not None:
     for i, obs in enumerate(observations):
       _ = numpyro.sample(
         name=f'observation_{i}',
         fn=dist.Normal(
           loc=media_contribution[
             obs['time_periods'],
             obs['channel']
           ].sum(axis=1),
           scale=0.2
         ),
         obs=jnp.array(obs['attributed_values'])
       )`
       
Is it used here now on line 411 to replace `jnp.einsum(media_einsum, media_transformed, coef_media)`? Here's how I'm thinking media_contribution is now used:

`  prediction = (
      intercept + coef_trend * trend ** expo_trend +
      seasonality * coef_seasonality +
      media_contribution)`
